modules/update-upgrading-cli.adoc

```diff
@@ -34,49 +34,54 @@ $ oc get clusterversion
 [source,terminal]
 ----
 NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
-version   4.8.13     True        False         158m    Cluster version is 4.8.13
+version   4.8.37     True        False         158m    Cluster version is 4.8.37
 ----
 
-. Review the current update channel information and confirm that your channel
-is set to `stable-4.9`:
+. View the available updates and note the version number of the update that
+you want to apply:
 +
 [source,terminal]
 ----
-$ oc get clusterversion -o json|jq ".items[0].spec"
+$ oc adm upgrade
 ----
 +
 .Example output
 [source,terminal]
 ----
-{
-  "channel": "stable-4.9",
-  "clusterID": "990f7ab8-109b-4c95-8480-2bd1deec55ff"
-}
+Cluster version is 4.8.37
+
+Upstream is unset, so the cluster will use an appropriate default.
+Channel: stable-4.9 (available channels: candidate-4.8, candidate-4.9, eus-4.10, eus-4.8, fast-4.8, fast-4.9, stable-4.8, stable-4.9)
+
+Recommended updates:
+VERSION IMAGE
+4.8.39  quay.io/openshift-release-dev/ocp-release@sha256:7b6225ae221d92645083a04ceae214974e121108c1edbc8b4985ba84f945f5d8
+4.8.41  quay.io/openshift-release-dev/ocp-release@sha256:5c551dfad2916d4f718b9e060392fc85e2832232351ecfeac2fbda57c62eee9d
+4.9.29  quay.io/openshift-release-dev/ocp-release@sha256:b04ca01d116f0134a102a57f86c67e5b1a3b5da1c4a580af91d521b8fa0aa6ec
+4.9.31  quay.io/openshift-release-dev/ocp-release@sha256:2a28b8ebb53d67dd80594421c39e36d9896b1e65cb54af81fbb86ea9ac3bf2d7
+4.9.32  quay.io/openshift-release-dev/ocp-release@sha256:ecdb6d0df547b857eaf0edb5574ddd64ca6d9aff1fa61fd1ac6fb641203bedfa
+4.9.33  quay.io/openshift-release-dev/ocp-release@sha256:fef77dfcc3e7c6ba175a4983816363511abc1783a0aac6993e74d05c8ab09496
+4.9.35  quay.io/openshift-release-dev/ocp-release@sha256:d617e6436691f002da49503aeeba84028846c50793115ee2ab6e040eb3713ce7
 ----
-+
-[IMPORTANT]
-====
-For production clusters, you must subscribe to a `stable-\*` or `fast-*` channel.
-====
 
-. View the available updates and note the version number of the update that
-you want to apply:
+. Based on your organization requirements, set the upgrade channel to `stable-4.9`, `fast-4.9`, or `eus-4.10`:
 +
 [source,terminal]
 ----
-$ oc adm upgrade
+$ oc adm upgrade channel <channel>
```

Comment:
openshift/oc#576 landed in the development branch and shipped with 4.9.  So folks using a 4.8 `oc` won't have the `channel` subcommand available.  That is narrowly acceptable, because we open this file [with][1]:

> Install the OpenShift CLI (`oc`) that matches the version for your updated version.

But I'm not sure how many folks are following along that carefully, so it might be worth a reminder here that we want them to be using a 4.9 `oc` for this pre-update channel change, even though they are currently running a 4.8 cluster.

[1]: https://github.com/openshift/openshift-docs/blame/d056c50234fe59ecff9b74f790ae2d10619baf23/modules/update-upgrading-cli.adoc#L19